modules/analytics-v2/src/views/full/index.tsx

```diff
@@ -85,38 +97,84 @@ export default class Analytics extends React.Component<{ bp: any }> {
     const startDate = moment.unix(this.state.startDate).toDate()
     const endDate = moment.unix(this.state.endDate).toDate()
     return (
-      <DateRangeInput
-        closeOnSelection={false}
-        formatDate={date => moment(date).format('MMMM Do YYYY')}
-        maxDate={new Date()}
-        parseDate={str => new Date(str)}
-        onChange={this.handleDateChange}
-        value={[startDate, endDate]}
-      />
+      <div className={style.filtersInputWrapper}>
+        <span className={style.inputLabel}>Date Range</span>
+        <DateRangeInput
+          closeOnSelection={true}
+          formatDate={date => moment(date).format('MM-DD-YYYY')}
+          maxDate={new Date()}
+          parseDate={str => new Date(str)}
+          onChange={this.handleDateChange}
+          value={[startDate, endDate]}
+        />
+      </div>
     )
   }
 
   render() {
-    if (this.isLoaded()) {
-      return (
-        <div>
+    if (!this.isLoaded()) {
+      return null
+    }
+
+    const { channels, shownSection, pageTitle, selectedChannel, showFilters } = this.state
+    const startDate = moment(moment.unix(this.state.startDate).toDate()).format('MMMM Do YYYY')
+    const endDate = moment(moment.unix(this.state.endDate).toDate()).format('MMMM Do YYYY')
+
+    return (
+      <Container>
+        <SidePanel>
+          <ul className={classnames(style.sideListList)}>
+            {sideList.map(item => (
+              <li
+                onClick={() => this.setState({ shownSection: item.key, pageTitle: item.label })}
+                key={item.key}
+                className={classnames(style.sideListItem, {
+                  [style.sideListItemSelected]: item.key === shownSection
+                })}
+              >
+                {item.label}
+              </li>
+            ))}
+          </ul>
```

Comment:
Can you use the existing component ItemList instead ? It's the same component we use in NLU and the CMS, so it provide a similar look & feel. Don't hesitate to update the UX or design if you want to make it look better https://github.com/botpress/botpress/blob/master/src/bp/ui-studio/src/web/components/Shared/Interface/index.tsx#L114